src/main/java/uk/gov/ch/oauth/Oauth2.java

```diff
@@ -201,4 +205,26 @@ private void regenerateSessionID(HttpServletResponse httpServletResponse) {
 
         httpServletResponse.addCookie(buildSessionCookie(session));
     }
+
+    public void invalidateSession(Session chSession, Store store) {
+        final Map<String, Object> sessionData = chSession.getData();
+        if (chSession.getSignInInfo().isSignedIn()) {
+            removeSessionInfo(sessionData, SIGN_IN_INFO);
+            removeZXSInfo(sessionData, store);
+        }
+    }
+
+    private void removeSessionInfo(Map<String, Object> sessionData, String signInfo) {
+        final Map<String, Object> signInInfo =
+                (Map<String, Object>) sessionData.get(signInfo);
```

Comment:
Don't need `signInfo` parameter if SIGN_IN_INFO is a constant. We can call the constant directly instead of passing into method.

Reply:
I have altered the code so that the constant is being directly called.